slynk/slynk.lisp

```diff
@@ -2966,11 +2974,19 @@ soon once non-ASDF loading is removed. (see github#134)")
 Receives a module name as argument and should return non-nil if it
 managed to load it.")
   (:method ((method (eql :slynk-loader)) module)
-    (funcall (intern "REQUIRE-MODULE" :slynk-loader) module))
+    (funcall (intern #.(if (eq :UPCASE (readtable-case *readtable*))
```

Comment:
Here I think we can use `uiop:intern*`, which can take symbols as arguments, to get a simpler solution: `(uiop:intern* '#:require-module '#:slynk-loader)`

For that matter, one could even do:

    (uiop:symbol-call '#:slynk-loader '#:require-module module)

Reply:
Is ASDF/UIOP already a dependency of this file?  I don't think so (may be wrong)  but I would avoid it.  I know Allegro ships with it (in recent versions), but not necessarily all SLY backends do.